### PR TITLE
Customizable pane width

### PR DESCRIPTION
**Pull request by WinXaito**
Add `width`, `minWidth` and `maxWidth` properties for an open `NavigationPane`.

Linked with #99 

## Pre-launch Checklist

- [x] I have run `dartfmt` on all changed files  
- [x] I have updated `CHANGELOG.md` with my changes  
- [x] ~~I have run "optimize/organize imports" on all changed files~~
- [x] ~~I have addressed all analyzer warnings as best I could~~
- [x] ~~I have added/updated relevant documentation~~
- [x] I have run `flutter pub publish --dry-run` and addressed any warnings

### CODE REVIEW

**Review by NicolaVerbeeck — APPROVED** [review]

**Review by bdlukaa — COMMENTED** [review]
Maybe we could have something for compact and top mode (height)?

**Comment by WinXaito** [comment]
image

image

Now can be edited like that...

File: example/lib/main.dart [hunk 1]
```diff
@@ -152,6 +152,11 @@ class _MyHomePageState extends State<MyHomePage> {
       pane: NavigationPane(
         selected: index,
         onChanged: (i) => setState(() => index = i),
+        size: NavigationPaneSize(
+          openWidth: MediaQuery.of(context).size.width / 5,
```
Comment (bdlukaa) [inline example/lib/main.dart:156]: Why to use this here?

Comment (WinXaito) [reply example/lib/main.dart:156]: Only for the demonstration of the capactity to have a dynamic width with a minimum and a maximum width.

**Review by bdlukaa — APPROVED** [review]

**Pull request merged**